### DISCUSSION TITLE
Adding a secondary text to webview title

### DISCUSSION THREAD

**Discussion by amitjoshi438** [microsoft/vscode-discussions#669] [Extension Development QnA] [answered]
Want to add "PREVIEW" text to title of webview view like below:

![image](https://github.com/microsoft/vscode-discussions/assets/54068463/27b2ee94-fce2-4f98-8733-1a2718f4668c)

**Comment by alefragnani** [comment]
Hi @amitjoshi438 ,

You are looking for the `TreeView.description` property. 

Once you create your `TreeView` using the `vscode.window.createTreeView` API, you can update the description whenever necessary.

```ts
    storageTreeView = vscode.window.createTreeView("projectsExplorerFavorites", { 
      treeDataProvider: this.storageProvider, 
      showCollapseAll: false } );
    ...
    storageTreeView.description = `${numberOfDisabledProjects} disabled / filtered by tags`;
```

Hope this helps

**Reply by amitjoshi438** [reply]
Thanks @alefragnani! I wanted to know if this is supported for Webview View title also?

**Reply by alefragnani** [reply, marked as answer]
Hi @amitjoshi438 ,

Sorry about that, I missed the _webview_ piece of your request. 

But, don't worry, it works almost the same way. The `vscode.WebviewView` interface has the same `description` property, so you just have to define it.

For instance, if you use the **webview-view-sample** project from the **vscode-extension-sample** repo and `webviewView.description = "some description" above this line, it will display that

![image](https://github.com/microsoft/vscode-discussions/assets/3781424/211a8f97-7484-4756-baa8-cf6327e00a7b)

https://github.com/microsoft/vscode-extension-samples/blob/672ac9c79bbfd41a05600a94e626466c4a598586/webview-view-sample/src/extension.ts#L38C13-L38C13

Hope this helps

**Reply by amitjoshi438** [reply]
Thanks a lot!
